plan/physical_plan_builder.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plan
+
+// Convert2PhysicalPlan implements Physical Convert2PhysicalPlan interface.
```

Comment:
Convert2PhysicalPlan is the interface of LogicalPlan. So as the following functions.